Makefile

```diff
@@ -12,6 +12,8 @@ OS := $(shell echo "$(GOOS)" | awk '{print toupper(substr($$0, 1, 1)) tolower(su
 SYMLINK_ARCH := $(if $(filter $(GOARCH),amd64),x86_64,$(if $(filter $(GOARCH),arm64),aarch64,$(ARCH)))
 POSTREST_ARCH_SUFFIX := $(if $(filter $(GOARCH),amd64),linux-static-x64.tar.xz,$(if $(filter $(GOARCH),arm64),ubuntu-aarch64.tar.xz,$(ARCH)))
 
+RUST_TARGET := $(if $(filter $(GOOS),linux),$(if $(filter $(GOARCH),amd64),x86_64-unknown-linux-gnu,$(if $(filter $(GOARCH),arm64),aarch64-unknown-linux-gnu)),$(if $(filter $(GOOS),darwin),$(if $(filter $(GOARCH),amd64),x86_64-apple-darwin,$(if $(filter $(GOARCH),arm64),aarch64-apple-darwin))))
```

Comment:
What it does:
It maps Go's GOOS and GOARCH variables to Rust target triplets:
If GOOS=linux:
```
GOARCH=amd64 → x86_64-unknown-linux-gnu
GOARCH=arm64 → aarch64-unknown-linux-gnu
```

If GOOS=darwin (macOS):
```
GOARCH=amd64 → x86_64-apple-darwin
GOARCH=arm64 → aarch64-apple-darwin
```
Why it's needed:
Go uses simple names like linux/amd64
Rust uses more specific target triplets like x86_64-unknown-linux-gnu
This variable converts between the two naming conventions
Example:
When you run make build GOOS=linux GOARCH=amd64
RUST_TARGET becomes x86_64-unknown-linux-gnu
The Rust build command becomes: cross build --release --target x86_64-unknown-linux-gnu
This ensures the Rust binary is built for the same platform as the Go binary.